### ISSUE TITLE
`article.content` only contains a part of the content

### ISSUE THREAD

**Issue by westlinkin** [luin/readability#100]
First of all, great great library! You've done a wonderful job here.

When use  [this url](http://www.etonline.com/edward-james-olmos-20th-anniversary-selena-ive-never-had-more-difficult-film-exclusive-89417), the result is wrong. The `article.content` only contains a part of the content, here the value: 

```
<div class="field field-paragraph field-paragraph--full field-type-text-long field-type-text-long--full"><p>“It was the saddest movie I've ever filmed, to be honest with you. I've never had a more difficult film to film,” Olmos lamented. “It was too close to the time when she was actually killed, it was only 13 months after when we were filming. Nobody wanted to film it, the parents didn't, we didn't, nobody wanted to. We'd rather she be alive. But we had to."</p></div>
```

If you click on the link, you'll see `article.content` only contains the first paragraph.

**Comment by wong2** [comment]
yes this is one of the biggest limitations of this lib: it doesn't work well on nested HTML structure:

![image](https://user-images.githubusercontent.com/321947/32259562-54e4cbfa-be8f-11e7-83b7-230ac9108dcb.png)

**Comment by raju1988** [comment]
Yes we are also facing same kind of issue. It won't return full content of html. It returns some random div from the page.
I used this link https://vnexpress.net/the-gioi/con-duong-lam-giau-bi-an-cua-ty-phu-au-dam-my-3967810.html
